src/components/WorkOrder/WorkOrderDetails.js

```diff
@@ -22,48 +20,39 @@ const WorkOrderDetails = ({
 }) => {
   const { user } = useContext(UserContext)
 
+  const allowedMenu = () => {
+    return WORK_ORDER_DROP_DOWN_MENU.filter((choice) => {
```

Comment:
What do you think about renaming this module and constant to remove the 'drop down' and 'menu'? That describes a UI implementation which could change. The module on its own just describes the various actions and permissions on them. What do you think of 'workOrderActions' or 'workOrderChoices'?